example/lib/pages/picture_in_picture_page.dart

```diff
@@ -50,12 +50,19 @@ class _PictureInPicturePageState extends State<PictureInPicturePage> {
           'betterPlayerEventType: ${event.betterPlayerEventType}, event.parameters: ${event.parameters.toString()}');
 
       if (event.betterPlayerEventType == BetterPlayerEventType.play) {
+        if (_willSwitchToPIPLayout) {
```

Comment:
These changes in files are not related to PR, update for testing (pip close when pausing video)

Reply:
Do you need to this modification to be merge? 
Not sure why this modification was needed but If it is just a test and not needed in our app, I think should not be in PR.

Reply:
Oh, each time I use this sample to test I always add this condition to test.

The case is in PIP mode if you press pause or play, app will close PIP (actually, it's implemented in `nfc` app with `isPIPMode` flag) so I think we should include it in PR.

Reply:
It will affect android. I will update condition (only for iOS)

Reply:
@HiroshiOshiro @noboru-i 
I updated the changes (refactoring names and handle like nfc app), these changes fix the bug pip close when pausing video.
Sorry if my expression makes you confused 🙏
https://github.com/dwango-nfc/betterplayer/pull/33/commits/cd5637c45171d97da0192dbe45cb0b5f527180a6